src/pocketmine/nbt/NBT.php

```diff
@@ -107,7 +107,13 @@ public static function getItemHelper(CompoundTag $tag){
 			return Item::get(0);
 		}
 
-		$item = Item::get($tag->id->getValue(), !isset($tag->Damage) ? 0 : $tag->Damage->getValue(), $tag->Count->getValue());
+		if($tag->id instanceof IntTag){
+			$item = Item::get($tag->id->getValue(), !isset($tag->Damage) ? 0 : $tag->Damage->getValue(), $tag->Count->getValue());
+		}else{ //PC item save format
```

Comment:
Not `elseif($tag->id instanceof StringTag)` for clarification? (And `else assert(false)` or `else throw new \RuntimeException` if IDEs complain)